verifiers/internal/gha/provenance_test.go

```diff
@@ -1182,3 +1185,115 @@ func Test_VerifyVersionedTag(t *testing.T) {
+				Signatures: []dsselib.Signature{
+					{
+						Sig:   "MEUCIQCLO33ZR7g+FaUKCXR3tncSjGv/mfDiGjPuk7NmHVryPQIgHhratRQrI/Kn4fgO9zK5vglyfrGwbhSJkuvnfUcD5UM=",
+						KeyID: "",
+					},
+				},
+			},
+			provenanceOpts: &options.ProvenanceOpts{
+				ExpectedBranch:         nil,
+				ExpectedTag:            nil,
+				ExpectedVersionedTag:   nil,
+				ExpectedDigest:         "caaadba2846905ac477c777e96a636e1c2e067fdf6fed90ec9eeca4df18d6ed9",
+				ExpectedSourceURI:      "github.com/enteraga6/slsa-lvl3-generic-provenance-with-bazel-example",
+				ExpectedBuilderID:      "https://github.com/slsa-framework/slsa-github-generator/.github/workflows/delegator_lowperms-generic_slsa3.yml@refs/tags/v1.7.0",
+				ExpectedWorkflowInputs: map[string]string{},
+			},
+			byob: true,
+
+			trustedBuilderIDName: "https://github.com/slsa-framework/slsa-github-generator/.github/workflows/delegator_lowperms-generic_slsa3.yml@refs/tags/v1.7.0",
+
+			expectedID: nil,
```

Comment:
nit: Remove empty lines
```suggestion
			byob: true,
			trustedBuilderIDName: "https://github.com/slsa-framework/slsa-github-generator/.github/workflows/delegator_lowperms-generic_slsa3.yml@refs/tags/v1.7.0",

			expectedID: nil,
```